src/com/google/enterprise/adaptor/prebuilt/FilterMimetypes.java

```diff
@@ -137,8 +137,6 @@ public void transform(Metadata metadata, Map<String, String> params) {
     if (supportedExplicit.contains(ct)) {
       log.log(Level.FINE, ct + "is explicitly supported");
       insertDecision(ct, TransmissionDecision.AS_IS.toString());
```

Comment:
I think we will be responding with explicit AS_IS for everything but first instance of a type.